projects/cmake/Readme.txt

```diff
@@ -4,10 +4,13 @@ cmake [-DCMAKE_BUILD_TYPE=Debug] [-DVEC4_OPT=On] [-DCRC_OPT=On] [-DX86_OPT=On] [
 
 -DCMAKE_BUILD_TYPE=Debug - optional parameter, if you want debug build. Default buid type is Release
 -DVEC4_OPT=On  - optional parameter. set it if you want to enable additional VEC4 optimization (can cause additional bugs).
+-DCRC_ARMV8=On - optional parameter. set it if you want to enable armv8 hardware CRC.
 -DCRC_OPT=On - optional parameter. set it to use xxHash to calculate texture CRC.
--DX86_OPT=On - optional parameter. set it if you want to enable additional X86 ASM optimization (can cause additional bugs).
 -DNEON_OPT=On - optional parameter. set it if you want to enable additional ARM NEON optimization (can cause additional bugs).
--DCRC_ARMV8=On - optional parameter. set it if you want to enable armv8 hardware CRC.
--DNOHQ=On - optional parameter. set to build without realtime texture enhancer library (GLideNHQ).
--DUSE_SYSTEM_LIBS=On - optional parameter. set to use system provided libraries for libpng and zlib.
--DMUPENPLUSAPI=On - required parameter. currently cmake build works only for mupen64plus version of the plugin.
+-DX86_OPT=On - optional parameter. set it if you want to enable additional X86 ASM optimization (can cause additional bugs).
+-DNOHQ=On - build without realtime texture enhancer library (GLideNHQ).
+-DMUPENPLUSAPI=On - currently cmake build works only for mupen64plus version of the plugin.
+-DUSE_SYSTEM_LIBS=On - set to use system provided libraries for libpng and zlib.
+-DODROID=On - set if you need to build on an Odroid board.
+-DVERO4K=On - set if you need to build on the OSMC Vero4k.
+
```

Comment:
Why so many changes when you just need to add two options and thus two lines of text?

Reply:
Sorry, I've not added to a commit before.
I thought I did just that, committed to master with the 2 lines at the bottom.  Simple enough as you say.
But then Git declared there was a merge conflict, and showed the lines all shuffled for some reason.  So having corrected the order again this is where we are.  I'm not proud of it, but there it is.
How would you like to proceed?

Reply:
Please take your original commit, which is without corrections in Readme.txt, and rebase it to current master. Then add these two lines to Readme.txt, update the commit and PR it there.